debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -962,8 +965,9 @@ protected String columnBlacklist() {
         return config.getString(COLUMN_BLACKLIST);
     }
 
-    protected int rowsFetchSize() {
-        return config.getInteger(ROWS_FETCH_SIZE);
+    public int getSnapshotFetchSize() {
```

Comment:
Could this be pulled up to `CommonConnectorConfig`? How about MySQL btw., I'd have assumed some change would be needed there, too?

Reply:
I moved the method to `CommonConnectorConfig` class. The MySQL connector has hack to reduce memory allocation (see https://issues.jboss.org/browse/DBZ-94). I moved the hint value to configuration.